### PR TITLE
[Admin] Country code uniquness validation

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | partially #4374
| License         | MIT
| Doc PR          |

### CODE REVIEW

File: features/localization/managing_countries/country_unique_code_validation.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    In order to avoid making mistakes when managing countries
+    As an Administrator
+    I want to be prevented from adding a new country with an existing code
+
+    Background:
+        Given the store operates in Norway
+        And I am logged in as administrator
+
+    @todo
+    Scenario: Trying to add new counrty with used code
```
Comment (TheMadeleine) [inline features/localization/managing_countries/country_unique_code_validation.feature]: `a new country` here and below, and you've got a typo :)

File: features/localization/managing_countries/country_unique_code_validation.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    As an Administrator
+    I want to be prevented from adding a new country with an existing code
+
+    Background:
+        Given the store operates in "Norway"
+        And I am logged in as administrator
+
+    @todo
+    Scenario: Trying to add a new country with used code
+        When I want to create new country
```
Comment (michalmarcinkowski) [inline features/localization/managing_countries/country_unique_code_validation.feature]: ~~create~~ **add**

Comment (pamil) [inline features/localization/managing_countries/country_unique_code_validation.feature]: **a** new country

**Comment by pjedrzejewski** [comment]
Thanks Łukasz!

File: features/localization/managing_countries/country_unique_code_validation.feature [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    As an Administrator
+    I want to be prevented from adding a new country with an existing code
+
+    Background:
+        Given the store operates in "Norway"
+        And I am logged in as administrator
+
+    @ui
+    Scenario: Trying to add a new country with used code
+        When I want to create new country
```
Comment (michalmarcinkowski) [inline features/localization/managing_countries/country_unique_code_validation.feature]: `I want to add a new country`?

**Pull request merged**